### DISCUSSION TITLE
Make an uncut event?

### DISCUSSION THREAD

**Discussion by XJY00** [zenorocha/clipboard.js#787] [Ideas]
Sometimes, 
I need to copy the necessary information before taking the jump other websites,
 so I use JS to choose to confirm or cancel the jump and copy, 
so I have encountered a problem, 
I can't cancel the copy.
The one I searched for can't do the perfect effect.
- [ ] clipboard.destroy();
Is there any good way to do it?
Ask the author to add this new feature or something.

**Comment by XJY00** [comment]
If anyone knows, please contact me.
xjy-00@outlook.com

**Comment by obetomuniz** [comment]
What about use `ClipboardJS.copy(’’)` when the user selected Cancel?

I know it's not a native approach, but would that work as a reset helper for your case?
